saleor/account/utils.py

```diff
@@ -77,3 +82,9 @@ def remove_staff_member(staff):
         staff.save()
     else:
         staff.delete()
+
+
+def requestor_is_staff_member(requestor: Union[User, "App"]):
+    """Return true if requestor is an active app or active staff user."""
+    is_staff = getattr(requestor, "is_staff", True)
```

Comment:
Why default `True` here? It would be safer to assume `False` when we are not sure the field `is_staff` is present.

Reply:
I assumed that, as `App` does not have `is_staff` field, so the `App` instance is always treated as a staff member. But maybe check with `isinstance` will be better.